lms/djangoapps/commerce/utils.py

```diff
@@ -46,8 +46,7 @@ def __init__(self):
 
     def is_enabled(self, user):
         """ Check if the user is activated, if the service is enabled and that the site is not a microsite. """
-        return (user.is_active and self.config.checkout_on_ecommerce_service and not
-                helpers.is_request_in_themed_site())
+        return user.is_active and self.config.checkout_on_ecommerce_service
```

Comment:
@e-kolpakov -- I think if we were to merge this line in, all microsites would begin using the Otto checkout process, correct?  This seems like something that we should avoid at the present time because we are not ready to enable Otto for the microsites running on edx.org.  I discussed with @mjfrey and @douglashall, and we are thinking that this should be a separate PR to master.  Alternatively, we could add a site_id to the commerce configuration model and that would allow for multiple microsites to control their own setups.  Thoughts?